### PR TITLE
Remove lodash fp/forms system fields

### PR DESCRIPTION
**Pull request by theodur**
## Description
Remove `lodash/fp` from forms system fields.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#1678

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms-system/src/js/fields/FileField.jsx [hunk 1]
```diff
@@ -146,21 +148,19 @@ class FileField extends React.Component {
 
   onAttachmentIdChange = (index, value) => {
     if (!value) {
-      this.props.onChange(
-        _.unset([index, 'attachmentId'], this.props.formData),
-      );
+      this.props.onChange(unset([index, 'attachmentId'], this.props.formData));
```
Comment (bkjohnson) [inline src/platform/forms-system/src/js/fields/FileField.jsx:151]: I think `unset` is fine - the alternative would be a `delete`, and that would lead to more refactoring in this function which is out of scope.

**Review by bkjohnson — COMMENTED** [review]
Sorry for all of the comments! I didn't leave one everywhere, but I think most of the `get` and `set` calls can be removed entirely.

File: src/platform/forms-system/src/js/fields/FileField.jsx [hunk 2]
```diff
@@ -283,7 +283,7 @@ class FileField extends React.Component {
           <ul className="schemaform-file-list">
             {files.map((file, index) => {
               const errors =
-                _.get([index, '__errors'], errorSchema) ||
+                get([index, '__errors'], errorSchema) ||
```
[the remark this answers is not in the document]
Comment (theodur) [reply src/platform/forms-system/src/js/fields/FileField.jsx:286]: That should work, it doesn't look like `errors` gets modified anywhere.

**Review by bkjohnson — APPROVED** [review]
Oops, I got the mutability of `lodash` and `lodash/fp` mixed up. Yeah, this is good as is.
